### PR TITLE
chore(merge): automate the PR merging process

### PR DESCRIPTION
**Pull request by esarbanis**

### CODE REVIEW

File: scripts/gh-pr-merge.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+function requestConfigInfo() {
+  var deferred = q.defer();
+  inquirer.prompt([
+    {
+      type: 'input',
+      name: 'prno',
+      message: 'Which PR number would you like to merge?'
+    },
+    {
+      type: 'list',
```
Comment (bekos) [inline scripts/gh-pr-merge.js]: Can you change the Yes/No questions to `type: 'confirm'`?

File: scripts/gh-pr-merge.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      choices: ['yes', 'no'],
+      filter: answer => answer === 'yes',
+      default: 0
+    }
+  ], function (answers) {
+    deferred.resolve({
+      prno: answers.prno,
+      append: answers.append,
+      test: answers.test
+    });
```
Comment (bekos) [inline scripts/gh-pr-merge.js]: You can just `deferred.resolve(answers);`.

**Comment by bekos** [comment]
@esarbanis This is really good stuff! :+1:  I would like though to make functions like `ghprb `, `runTests` etc return promises so we can separate the whole process into small individual tasks.

**Comment by bekos** [comment]
@esarbanis FYI, seems to be a problem with the commit history of this PR.

**Comment by esarbanis** [comment]
@tbekos Just dealt with it :)

**Comment by bekos** [comment]
@esarbanis There are some really cool stuff in there :-) I kept what is absolutely necessary and tedious when merging a PR, cleaned up and landed it as 0c1e99d.

Feel free to open new PRs for the missing stuff.  Thx :+1:

**Pull request closed without merging**